libs/domains/clusters/feature/src/lib/hooks/use-upgrade-cluster/use-upgrade-cluster.ts

```diff
@@ -0,0 +1,21 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query'
+import { mutations } from '@qovery/domains/clusters/data-access'
+import { queries } from '@qovery/state/util-queries'
+
+export interface UseUpgradeClusterProps {
+  organizationId: string
+}
+
+export function useUpgradeCluster({ organizationId }: UseUpgradeClusterProps) {
+  const queryClient = useQueryClient()
+
+  return useMutation(mutations.upgradeCluster, {
+    onSuccess(_, { clusterId }) {
+      queryClient.invalidateQueries({
```

Comment:
You need to invalid `clusters.listStatuses` as well because it's used in the clusters list

```
     queryClient.invalidateQueries({
        queryKey: queries.clusters.listStatuses({ organizationId }).queryKey,
      })
```

Reply:
Fixed https://github.com/Qovery/console/pull/1655/commits/1ccd6de563bdf4a3e01ba0572952c5c479f2b085